src/Core/Framework/Plugin/Command/PluginListCommand.php

```diff
@@ -18,6 +18,31 @@ class PluginListCommand extends Command
 {
     protected static $defaultName = 'plugin:list';
 
+    /**
+     * @var int
+     */
+    protected $installed;
+
+    /**
+     * @var int
+     */
+    protected $upgradeable;
+
+    /**
+     * @var int
+     */
+    protected $active;
+
+    /**
+     * @var int
+     */
+    protected $total;
+
+    /**
+     * @var string[]
+     */
+    protected $availableFormats;
```

Comment:
```suggestion
    private $availableFormats = ['table', 'json'];
```

with this, the part in the constructor is of course obsolete

Reply:
True, but why private?

Reply:
why does it need to be protected?

Reply:
@mitelg good point, that's not needed either. I'll change it to public

Reply:
I hope you mean private? :smile: